CONP_DATS_fields.md

```diff
@@ -1,51 +1,52 @@
+|Dimensions|dimensions|**RECOMMENDED**| The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|
+|Identifier|identifier|| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Identifier|Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying an entity locally to a system or globally.|
+|Identifier|Identifier -> Identifier Source|**RECOMMENDED**| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Contact|extraProperties - contact|**RECOMMENDED**| Contact information for this dataset.|Provide contact information (name and email address) of the person responsible for the dataset.| 
+|Logo|logo|**RECOMMENDED**| Link to or provide a logo to display on your resource landing page.|Link to an URL for the logo or local filename containing the logo.
+|Dates|dates|**OPTIONAL**| Relevant dates for the dataset. If you provide a date, it must come with a description of the date.|Relevant dates for the dataset. If you provide a date, it must come with a description of the date (i.e.: first data collection, last data collection, date of first publication, ...). |<!--will later choose from a pulldown list--> 
+|Produced By|producedBy|**OPTIONAL**| Process which generated a given dataset.|Process which generated a given dataset.|
+|Is About|isAbout|**OPTIONAL**| Entities (biological entity, taxonomic information, disease, molecular entity, anatomical part, treatment) associated with this dataset.| |
+|Acknowledges|acknowledges|**OPTIONAL**| Grant(s) which funded and supported the work reported by the dataset.| Grant(s) which funded and supported the work reported by the dataset.|
+|Spatial Coverage|spatialCoverage|**OPTIONAL**| The geographical extension and span covered by the dataset.|The geographical extension and span covered by the dataset.|
 
 The DATS dataset schema can be found [here](https://github.com/CONP-PCNO/schema/blob/master/dataset_schema.json), and the `DATS.json` file from the visual-working-memory dataset [here](https://github.com/conpdatasets/ds001634/blob/master/DATS.json) can be used as a template. A graphic interface allowing users to fill in fields online is under development [here](https://dats-creator.herokuapp.com/).
```

Comment:
At this point I suggest cutting the last sentence here and replacing it with a link to the actual GUI.

Reply:
Excellent point!